### PR TITLE
Priyanshi/rum home page

### PR DESCRIPTION
**Pull request by priyanshi-gupta**
### What does this PR do?

Adds context about RUM capabilities on home page

### Preview

https://docs-staging.datadoghq.com/priyanshi/rum_home_page/real_user_monitoring/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by hdelaby — APPROVED** [review]
Great job!!! :tada: 2 minor comments:
- for some reasons the language icons look zoomed-in and pixels are visible
- we can find better links for the further reading section

File: content/en/real_user_monitoring/_index.md [hunk 1]
```diff
@@ -21,21 +21,54 @@ further_reading:
...
-  {{< nextlink href="/real_user_monitoring/android">}}<u>Android Monitoring</u>: Create an application by configuring the Android SDK.{{< /nextlink >}}
-  {{< nextlink href="/real_user_monitoring/dashboards">}}<u>Dashboards</u>: Discover all data collected out of the box within an out of the box Dashboard.{{< /nextlink >}}
-{{< /whatsnext >}}
-{{< whatsnext desc="Explore your RUM events:">}}
-  {{< nextlink href="/real_user_monitoring/explorer/">}}<u>RUM Search</u>: Search through your page views.{{< /nextlink >}}
-  {{< nextlink href="/real_user_monitoring/explorer/analytics">}}<u>RUM Analytics</u>: Get insights from all your events.{{< /nextlink >}}
-{{< /whatsnext >}}
+* **Performance**: Track the performance of web pages, mobile application screens, user actions, network requests, and your front-end code
+* **Error Management**: Monitor the ongoing bugs and issues and track them over time and versions
+* **Analytics / Usage**: Understand who is using your application (country, device, OS…), monitor individual users journeys, and analyze how users interact with your application (most common page visited, clicks, interactions, feature usage…)
```
Comment (apigirl) [inline content/en/real_user_monitoring/_index.md:35]:
```suggestion
* **Analytics / Usage**: Understand who is using your application (country, device, OS), monitor individual users journeys, and analyze how users interact with your application (most common page visited, clicks, interactions, feature usage…)
```

File: content/en/real_user_monitoring/_index.md [hunk 2]
```diff
@@ -21,21 +21,54 @@ further_reading:
...
+* **Support**: Retrieve all of the information related to one user session to troubleshoot an issue (session duration, pages visited, interactions, resources loaded, errors…)
```
Comment (apigirl) [inline content/en/real_user_monitoring/_index.md:36]:
```suggestion
* **Support**: Retrieve all of the information related to one user session to troubleshoot an issue (session duration, pages visited, interactions, resources loaded, errors)
```

File: content/en/real_user_monitoring/_index.md [hunk 3]
```diff
@@ -21,21 +21,54 @@ further_reading:
...
+
+### Out of the Box Dashboards
+
+Analyze information about your user journeys, performance, network requests, and errors collected automatically with [out of the box dashboards][1].
+
+{{< img src="real_user_monitoring/dashboards/rum_dashboard.png" alt="RUM dashboard" >}}
+
+### RUM Explorer and Analytics
+
+Slice and dice user sessions by segments such as checking when latency impacts your premium customers with [customizable analytics widgets][2]. Explore, save views and create monitors on your customized searches.
```
Comment (apigirl) [inline content/en/real_user_monitoring/_index.md:56]:
```suggestion
View user sessions in segments, such as checking when latency impacts your premium customers with [customizable analytics widgets][2]. Explore, save views, and create monitors on your customized searches.
```

File: content/en/real_user_monitoring/_index.md [hunk 4]
```diff
@@ -21,21 +21,54 @@ further_reading:
...
+
+{{< img src="real_user_monitoring/explorer/analytics/rum_analytics.gif" alt="RUM Analytics"  >}}
+
+### Seamless integration with Logs, APM and Profiler
+
+View your [backend traces, logs, infrastructure metrics][3] to line of code impacting your application performance that correspond  to user experiences and reported issues.
```
Comment (apigirl) [inline content/en/real_user_monitoring/_index.md]:
```suggestion
View your [backend traces, logs, infrastructure metrics][3] to line of code impacting your application performance that correspond to user experiences and reported issues.
```

File: content/en/real_user_monitoring/_index.md [hunk 5]
```diff
@@ -21,21 +21,54 @@ further_reading:
...
+
+{{< img src="real_user_monitoring/connect_rum_and_traces/rum_apm_logs.png" alt="RUM and APM">}}
+
+### Error tracking and crash reporting
+
+Get automated alerts on outliers and groups of errors, timeouts, crashes to significantly reduce your MTTR with [Error Tracking][4].
```
Comment (apigirl) [inline content/en/real_user_monitoring/_index.md:68]:
```suggestion
Get automated alerts on outliers and groups of errors, timeouts, and crashes to significantly reduce your MTTR with [Error Tracking][4].
```

**Review by apigirl — COMMENTED** [review]
just some very small updates

File: content/en/real_user_monitoring/_index.md [hunk 6]
```diff
@@ -21,21 +30,54 @@ further_reading:
...
-  {{< nextlink href="/real_user_monitoring/dashboards">}}<u>Dashboards</u>: Discover all data collected out of the box within an out of the box Dashboard.{{< /nextlink >}}
-{{< /whatsnext >}}
-{{< whatsnext desc="Explore your RUM events:">}}
-  {{< nextlink href="/real_user_monitoring/explorer/">}}<u>RUM Search</u>: Search through your page views.{{< /nextlink >}}
-  {{< nextlink href="/real_user_monitoring/explorer/analytics">}}<u>RUM Analytics</u>: Get insights from all your events.{{< /nextlink >}}
-{{< /whatsnext >}}
+* **Performance**: Track the performance of web pages, mobile application screens, user actions, network requests, and your front-end code
+* **Error Management**: Monitor the ongoing bugs and issues and track them over time and versions
+* **Analytics / Usage**: Understand who is using your application (country, device, OS), monitor individual users journeys, and analyze how users interact with your application (most common page visited, clicks, interactions, feature usage…)
+* **Support**: Retrieve all of the information related to one user session to troubleshoot an issue (session duration, pages visited, interactions, resources loaded, errors)
```
Comment (kayayarai) [inline content/en/real_user_monitoring/_index.md:36]:
```suggestion
* **Performance**: Track the performance of web pages, mobile application screens, user actions, network requests, and your front-end code.
* **Error Management**: Monitor the ongoing bugs and issues and track them over time and versions.
* **Analytics / Usage**: Understand who is using your application (country, device, OS), monitor individual users journeys, and analyze how users interact with your application (most common page visited, clicks, interactions, feature usage).
* **Support**: Retrieve all of the information related to one user session to troubleshoot an issue (session duration, pages visited, interactions, resources loaded, errors).
```

File: content/en/real_user_monitoring/_index.md [hunk 7]
```diff
@@ -21,21 +30,54 @@ further_reading:
...
+
+## Getting started
+
+Select your application type to start collecting RUM data:
+
+{{< partial name="rum/rum-getting-started.html" >}}
+</br>
+## Explore Datadog RUM
+
+### Out of the Box Dashboards
```
Comment (kayayarai) [inline content/en/real_user_monitoring/_index.md:48]:
```suggestion
### Out of the box dashboards
```

File: content/en/real_user_monitoring/_index.md [hunk 8]
```diff
@@ -21,21 +30,54 @@ further_reading:
...
+
+### RUM Explorer and Analytics
+
+View user sessions in segments, such as checking when latency impacts your premium customers with [customizable analytics widgets][2]. Explore, save views, and create monitors on your customized searches.
+
+{{< img src="real_user_monitoring/explorer/analytics/rum_analytics.gif" alt="RUM Analytics"  >}}
+
+### Seamless integration with Logs, APM and Profiler
+
+View your [backend traces, logs, infrastructure metrics][1] to line of code impacting your application performance that correspond  to user experiences and reported issues.
```
Comment (kayayarai) [inline content/en/real_user_monitoring/_index.md:62]:
```suggestion
View your [backend traces, logs, infrastructure metrics][3] down to the exact line of code impacting your application performance, corresponding to user experiences and reported issues.
```

File: content/en/real_user_monitoring/_index.md [hunk 9]
```diff
@@ -21,21 +30,54 @@ further_reading:
...
 
 {{< partial name="whats-next/whats-next.html" >}}
+
+[1]:/real_user_monitoring/dashboards
+[2]:/real_user_monitoring/explorer/analytics
+[4]:/real_user_monitoring/error_tracking
+
+
+
+[1]: real_user_monitoring/connect_rum_and_traces
```
Comment (kayayarai) [inline content/en/real_user_monitoring/_index.md:83]:
```suggestion

```

File: content/en/real_user_monitoring/_index.md [hunk 10]
```diff
@@ -21,21 +30,54 @@ further_reading:
...
+
+{{< img src="real_user_monitoring/error_tracking/rum_errors.gif" alt="RUM error tracking">}}
+
 
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
+
+[1]:/real_user_monitoring/dashboards
+[2]:/real_user_monitoring/explorer/analytics
```
Comment (kayayarai) [inline content/en/real_user_monitoring/_index.md:78]:
```suggestion
[2]:/real_user_monitoring/explorer/analytics
[3]: /real_user_monitoring/connect_rum_and_traces
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Thanks! Some edits and a broken link fix

**Review by apigirl — APPROVED** [review]

**Pull request merged**